### DISCUSSION TITLE
Removing `str deunicode`

### DISCUSSION THREAD

**Discussion by kaathewisegit** [nushell/nushell#13677] [General]
The crate this command uses, [`deunicode`](https://lib.rs/crates/deunicode) only has 170 lines of code.  It, however, bundles 170KB worth of unicode data with it.  I think this functionality is better suited for a plugin and that base Nushell shouldn't bundle it.

@fncred what do you think?

**Comment by fdncred** [comment]
I added it because I needed it for a work thing with nushell. I figured it was handy for other people. I'd prefer to leave it for now but I'm sure others will have a different opinion.

**Reply by sholderbach** [reply]
For those missing it now, the plugin version is available with https://github.com/fdncred/nu_plugin_strutils

**Reply by kaathewisegit** [reply]
And there's also `decode unicode` in my [endecode](https://codeberg.org/kaathewise/nu-plugin/src/branch/trunk/endecode) plugin.

**Comment by sholderbach** [comment]
To me that sounds like something for a plugin for stabilization reasons. There is no internationally accepted spec for that and the underlying mapping is free to change, for a plugin that could version independently from main Nushell, for main Nushell changing behavior for output of commands should be considered a breaking change if not given some very good reasons.

There may be some reason to use this crate for tabcompletion with internatioalized input where it is easier for the user to match on ANSI compared to e.g. CJK input but interactive use has a looser stabilization requirement compared to a command. And I would wager that this issue is way to niche to be part of the core nushell.
